tests/test_cli/test_list.py

```diff
@@ -174,20 +174,36 @@ def teardown_class(cls):
             pass
 
 
-@mock.patch("aea.cli.common.try_to_load_agent_config")
 class ListContractsCommandTestCase(TestCase):
```

Comment:
I don't know if it has anything to do with the changes applied to this class, but the test `TestListSkills::test_correct_output` is now failing in CI.

Reply:
Weird issue, I see that. Previous time `test_search` failed and this passed. And locally everything passes. Will find out.